code/game/objects/items/weapons/material/shield_vr.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/weapon/material/sword/shield
+	name = "makeshift shield"
+	desc = "A makeshift shield"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "riot"
+	force_divisor = 0.07
+	sharp = FALSE
+	attack_verb = list ("attacked", "bashed", "smacked", "bonked", "spanked")
+	w_class = ITEMSIZE_LARGE
+
+/obj/item/weapon/material/sword/handle_shield(mob/user, var/damage, atom/damage_source = null, mob/attacker = null, var/def_zone = null, var/attack_text = "the attack")
```

Comment:
overwrites shield handling for all material swords and subtypes, not just the shield, due to inheritance; consider extending to `/obj/item/weapon/material/sword/shield/handle_shield`. or repath it

Reply:
also at first glance doesn't have blockchance scaling with any stat from the material, just the flat `prob(50)` in the line below